tools/make-tarball

```diff
@@ -15,24 +15,27 @@ Usage: ${0##*/} [revision]
     options:
       -h | --help		print usage
       -o | --output FILE	write to file
+           --version VERSION	Override the version used in the tarball
```

Comment:
maybe just:

    set the version used in the tarball.  Default value is determined with 'git describe'.